locales/en.json

```
@@ -190,7 +190,19 @@
             "invalid_year": "The card's expiration year is invalid.",
             "invalid_month": "The card's expiration month is invalid.",
             "invalid_cvv": "The card's CVV is invalid.",
-            "expired": "The card's expiration date is passed"
+            "expired": "The card's expiration date is passed.",

```

I think this would be better:

```suggestion
            "expired": "The card is expired.",
```